fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -826,12 +826,16 @@ private void appendInnerSelect(StringBuilder whereClauseSegment, QueryParameter
                 for (int componentNum = 1; componentNum <= components.size(); componentNum++) {
                     String alias = chainedParmVar + "_p" + componentNum;
                     QueryParameter component = components.get(componentNum - 1);
+                    String tableName = QuerySegmentAggregator.tableName(resourceTypeName, component);
```

Comment:
Might be worth a comment noting the issue number and the fact that composites still use the old token values table (for now)

Reply:
Done.